DataTable.svelte

```diff
@@ -191,6 +201,14 @@
       currentPageIndex = lastPageIndex
     }
   }
+
+  async function handleChangePagination(value: string) {
+    updatePageSize(Number(value))
+    await onChangeCurrentPageIndex?.(lastPageIndex)
```

Comment:
Why does it jump to the last page?
Is there a reason?

I thought the first page might be good.

Reply:
yes you are right I just changed that

Reply:
Thank you for the change!
Sorry, I'm on vacation until May 8th.
Please wait for another person's review